core/iwasm/fast-jit/fe/jit_emit_control.c

```diff
@@ -832,15 +865,51 @@ jit_compile_op_br(JitCompContext *cc, uint32 br_depth, uint8 **p_frame_ip)
            && handle_next_reachable_block(cc, p_frame_ip);
 }
 
+static JitFrame *
+jit_frame_clone(const JitFrame *jit_frame)
+{
+    JitFrame *jit_frame_cloned;
+    uint32 max_locals = jit_frame->max_locals;
+    uint32 max_stacks = jit_frame->max_stacks;
+    uint32 total_size;
+
+    total_size = (uint32)(offsetof(JitFrame, lp)
+                          + sizeof(*jit_frame->lp) * (max_locals + max_stacks));
```

Comment:
I don't have any opinion about the code. But just want to clarify that in a frame, all we actually care about is data in both `locals` and `stack`, right?

Reply:
No, not only locals and stacks, but also other info, e.g. frame_sp, virtual fixed registers, e.g. memory/table related registers. Since when clear_values at L972, most of the frame registers are set 0, we need to backup them firstly and then restore them after translating the dst basic block.